### PR TITLE
Split out Autotune to 3 docs

### PR DESCRIPTION
**Pull request by tim-statsig**
No changes to text

### CODE REVIEW

File: docs/autotune/setup.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+3. Provide the variants that you want to test in the Autotune experiment.  Each variant needs a name, and a corresponding JSON value.  The variant that's listed as Control/Default will be returned when the Autotune experiment is not running.
+
+![image](https://user-images.githubusercontent.com/1315028/131385189-5f0c1d93-ba87-4159-8995-3c30991587a0.png)
+
+4. Select the success event to optimize for as shown below.  You can further specify an optional [event value](/guides/logging-events).
+
+![image](https://user-images.githubusercontent.com/1315028/131385239-5a76d253-022b-457e-a370-f9ee7ce566a1.png)
+
+There are a few parameters you can specify:
+- Exploration Window - The initial time period where Autotune will equally split the traffic.  This is useful for noisy or temporal metrics where hourly swings in data can bias Autotune's initial measurements.
```
Comment (tore-statsig) [inline docs/autotune/setup.md:20]: Is it documented somewhere that autotune updates hourly?

File: docs/autotune/setup.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Attribution Window - The maximum duration between the exposure and success event that counts as a success.  We recommend 1 hr for most applications, but adjust accordingly if you expect the success event to lag the exposure event by several hours.
+- Winner Threshold - The threshold at for when Autotune will declare a winner and stop collecting data.  Setting a lower number will result in faster decisions but increases the probability of making suboptimal decisions (picking the wrong winner).
```
Comment (tore-statsig) [inline docs/autotune/setup.md:22]: "The threshold at for when" -> for which?

File: docs/autotune/setup.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Autotune
+sidebar_label: Setup
+slug: /autotune/setup
+---
+
+## How to set up Autotune
+
+1. To create a new Autotune experiment, navigate to the Autotune section on the Statsig console: https://console.statsig.com/
```
Comment (tore-statsig) [inline docs/autotune/setup.md:9]: To create a new Autotune experiment, navigate to the [Autotune section on the Statsig console](https://console.statsig.com/autotune).

You can link it like this :)

**Review by tore-statsig — APPROVED** [review]

File: docs/autotune/setup.md (hunk 3 above)
Comment (tim-statsig) [reply docs/autotune/setup.md:9]: 😮

**Pull request merged**